### DISCUSSION TITLE
Can UVR5 output 24 bit audio?

### DISCUSSION THREAD

**Discussion by robgronotte** [Anjok07/ultimatevocalremovergui#1027] [Q&A]
I have used this in the past with 16 bit CD quality audio and it works great.  I just tried running a 24/48 WAV file, and it worked fine, but the output is 16/44.1.  Is there any way to get the output in the original sample rate?

**Comment by ghost** [comment]
You can change the output bitdepth in settings:
(but you can't change sample rate as models were all trained on 44100hz audio)
![image](https://github.com/Anjok07/ultimatevocalremovergui/assets/6044199/4a9ab923-23c2-4854-9cd4-d16fab5768e8)

**Reply by robgronotte** [reply]
Thanks much!  Don't know how I missed that.  It would be nice if it could handle higher sample rates though.
